.circleci/config.yml

```diff
@@ -77,23 +64,21 @@ jobs:
       - checkout
       - *cache_save_git
       - *cache_restore_bundler
-      - *cache_restore_homebrew
       - *set_ruby
       - run:
           name: debug | ruby version
           command: |
             ruby -v
+      - run:
+          name: debug | brew version
+          command: |
+            brew -v
```

Comment:
I added this to track any future issues related to homebrew... There was no info about this in CircleCI's page

https://circle-macos-docs.s3.amazonaws.com/image-manifest/v4250/index.html